packages/forms-web-app/src/controllers/appeal-householder-decision/list-of-documents.js

```diff
@@ -22,17 +22,17 @@ const postListOfDocuments = async (req, res) => {
 		if (req.body['save-and-return'] !== '') {
 			const appeal = req.session.appeal;
 
-			const usingV2Form = await isFeatureActive(FLAG.APPEAL_FORM_V2, appeal.lpaCode);
+			const lpa = await getDepartmentFromId(appeal.lpaCode);
+			const lpaCode = lpa.lpaCode ?? (await getLPAById(lpa.id)).lpaCode; // fallback to lookup in case cached lpa doesn't have code
```

Comment:
this is brain pain 🤕 what is `appeal.lpaCode`? I guess not an lpaCode? 😅

Reply:
yes, yes it is

in the db it's:
	"lpa19CD" : "E6999999",
	"lpaCode" : "Q9999",
but on v1 appeals it's lpaCode = E6999999

and has been like that for years